openedx/core/djangoapps/enrollments/urls.py

```diff
@@ -29,4 +30,5 @@
             EnrollmentCourseDetailView.as_view(), name='courseenrollmentdetails'),
     path('unenroll/', UnenrollmentView.as_view(), name='unenrollment'),
     path('roles/', EnrollmentUserRolesView.as_view(), name='roles'),
+    path('enrollment_allowed', EnrollmentAllowedView.as_view(), name='courseenrollmentallowed'),
```

Comment:
```suggestion
    path('enrollment_allowed/', EnrollmentAllowedView.as_view(), name='courseenrollmentallowed'),
```

We should add a trailing slash to keep this consistent with the other URLs.